pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -378,7 +378,11 @@ func (ca *changeAggregator) Start(ctx context.Context) {
 	if ca.knobs.OverrideExecCfg != nil {
 		execCfg = ca.knobs.OverrideExecCfg(execCfg)
 	}
-	ca.targets, err = AllTargets(ctx, ca.spec.Feed, execCfg)
+	targetTS := ca.spec.SchemaTS
```

Comment:
Considering the number of times we do this if statement, could we define a `GetSchemaTS() hlc.Timestamp` receiver on `ChangeAggregatorSpec`?

ditto for `ChangeFrontierSpec`